fetcher.go

```diff
@@ -232,22 +227,35 @@ func (p *pool) fetchResource(ctx context.Context, from string, resource string,
 			}
 		}
 	}
+	req.Header.Add("User-Agent", "bifrost-"+os.Getenv(EnvironmentKey))
```

Comment:
don't over-ride the provided user agent - there's already a more specific user agent set on the client that bifrost passes in and this will over-write it.

Reply:
@willscott Can we append this to the existing `UserAgent` that the bifrost client has on it ? I believe L1s need this to be able to bifurcate metrics etc on their side based on environment.

Reply:
appending is okay, sure

Reply:
or you could set environment as another header, right?

Reply:
Appended to existing